ui/src/components/NavHeader/index.tsx

```diff
@@ -22,7 +22,11 @@ const NavHeader: React.FC = () => {
           </Toolbar.Item>
           <Toolbar.Item>
             <Navbar.Divider />
-            <HelpText className="ml-3 text-white">v1.0.0</HelpText>
+            <HelpText className="ml-3 text-white">
+              <a target='_blank' href='https://github.com/mergestat/fuse' rel='noopener noreferrer'>
+                <span className='cursor-pointer hover_text-blue-600'>beta</span>
```

Comment:
```suggestion
                <span>beta</span>
```

The `a` should already have a `cursor: pointer`. Also, I suggest removing the hover state to make it more subtle

Reply:
Done [8c39552](https://github.com/mergestat/fuse/pull/139/commits/8c39552603980dece9f39a22a54339b393ae1fed)